File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int product = 1;
        int[] output = new int[nums.length];
        for(int i=0;i<nums.length;i++){
            for(int j =0;j<nums.length;j++){
                if(i == j){
                    product +=0;
                }
                else{
                    product *= nums[j];
                }
            }
            output[i] = product;
            product =1;
        }
        return output;
    }
}  

File: Data Structures & Algorithms/special-array-i/submission-0.py
class Solution:
    def isArraySpecial(self, nums: List[int]) -> bool:
        if len(nums) <= 1:
            return True
        for i in range(len(nums)):
            if i == len(nums) - 1:
                if nums[i] % 2 ==  nums[i - 1]  % 2:
                    return False

            else:
                if (nums[i] % 2) ==  (nums[i + 1]  % 2):
                    return False

        return True